source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx

```diff
@@ -1,29 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <materialx version="1.38">
-  <!-- Implementation overrides for ESSL -->
+  <!--
```

Comment:
Is this file necessary at all? it is completely empty

Reply:
Just a stub, in case something gets added here later.

Reply:
I would remove it. No future proofing . Then we don't need to answer the question of the copyright either 😬